src/Compilers/CSharp/Portable/Compilation/AwaitExpressionInfo.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.CodeAnalysis.CSharp
     /// <summary>
     /// Structure containing all semantic information about an await expression.
     /// </summary>
+    // https://github.com/dotnet/roslyn/issues/75960: Add runtime async info
```

Comment:
I think it would be better to link to a dedicated issue rather than the test plan. I see that we have no other reference to the test plan in code, except this one